components/blocks/ModalButton/ModalButton.tsx

```diff
@@ -41,12 +41,12 @@ export const ModalB = ({ items, align = 'left', className = '' }) => {
           .filter(Boolean)
           .join(' ')}
       >
-        {items?.map((item) => {
+        {items?.map((item, index) => {
           const { color, label, icon, modal, shape } = item;
 
           return (
             <ModalButton
-              key={label}
+              key={sanitizeLabel(label) || modal || `modal-button-${index}`}
```

Comment:
Same here - just use the same key for consistency

```suggestion
              key={`modal-button-${index}`}
```